tasks/Dockerfile

```diff
@@ -65,6 +64,8 @@ RUN dnf -y update && \
 
 COPY cockpit-tasks install-service webhook github_handler.py /usr/local/bin/
 
+RUN pip3 install pika
```

Comment:
Please add a comment why the python3-pika RPM doesn't work. Also, please move this into the previous `RUN` block to avoid even more docker image layers. It semantically fits well into the "package install" layer.